material-about-this-book/drip-summary-300-words.md

```diff
@@ -0,0 +1,26 @@
+_Development Research in Practice_ is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+The Handbook leads the reader through a complete empirical research project,
```

Comment:
```suggestion
The handbook leads the reader through a complete empirical research project,
```

Reply:
should be capitalized because is reference to this particular Handbook

Reply:
ok.